owncloud/indico_owncloud/templates/add_owncloud_files.html

```diff
@@ -0,0 +1,62 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_row %}
+
+{%- block content %}
+    {{ form_header(form, id='attachment-owncloudfilepicker-form', classes=('full-width')) }}
+    {{ form_row(form.owncloud_filepicker, skip_label=true) }}
+    {{ form_rows(form, fields=('folder', 'protected', 'acl')) }}
+    {{ protection_message | safe }}
+    {% call form_footer(attach_form) %}
+        <input class="i-button big highlight" type="submit" value="{% trans %}Add{% endtrans %}">
+        <button class="i-button big" data-button-back>{% trans %}Cancel{% endtrans %}</button>
+    {% endcall %}
+    <script>
+        global.messageIfFolderProtected = function messageIfFolderProtected(
```

Comment:
It looks like it is already in `global` if I remove it from here (the core is adding it from `forms.js`). Would it be OK to assume it will stay there?

Reply:
yes, that's the point of it being global ;)